app/src/lib/stores/app-store.ts

```diff
@@ -6098,15 +6100,30 @@ export class AppStore extends TypedBaseStore<IAppState> {
       return
     }
 
+    const { parent, owner, name, htmlURL } = gitHubRepository
+    const isForkContributingToParent =
+      isRepositoryAForkContributingToParent(repository)
+
+    const baseForkPreface =
+      isForkContributingToParent && parent !== null
+        ? `${parent.owner.login}:${parent.name}:`
+        : ''
     const encodedBaseBranch =
       baseBranch !== undefined
-        ? encodeURIComponent(baseBranch.nameWithoutRemote) + '...'
+        ? baseForkPreface +
+          encodeURIComponent(baseBranch.nameWithoutRemote) +
+          '...'
         : ''
-    const encodedCompareBranch = encodeURIComponent(
-      compareBranch.nameWithoutRemote
-    )
+
+    const compareForkPreface = isForkContributingToParent
+      ? `${owner.login}:${name}:`
+      : ''
+
+    const encodedCompareBranch =
+      compareForkPreface + encodeURIComponent(compareBranch.nameWithoutRemote)
+
     const compareString = `${encodedBaseBranch}${encodedCompareBranch}`
-    const baseURL = `${gitHubRepository.htmlURL}/pull/new/${compareString}`
+    const baseURL = `${htmlURL}/pull/new/${compareString}`
```

Comment:
I found that the existing create pull request on dotcom path that does not pass in a base branch will always open to dotcom with a base branch of the parent repository. This surprised kind of because in that scenario this URL created here for my desktop fork will be `https://github.com/tidy-dev/desktop/pull/new/tidy-dev:desktop:Hello`., but, GitHub redirects to `https://github.com/desktop/desktop/compare/development...tidy-dev:desktop:Hello?expand=1`

However, if we were to pass a base branch in and apply the fork preface as this PR does - `https://github.com/tidy-dev/desktop/pull/new/tidy-dev:desktop:development...tidy-dev:desktop:Hello`; it will correctly go to `https://github.com/tidy-dev/desktop/compare/tidy-dev:desktop:development...tidy-dev:desktop:Hello?expand=1`

I may open a PR if this one goes through to do that because I think that is the expected behavior when not contributing to the parent.